analyzers/tests/SonarAnalyzer.UnitTest/TestCases/ReturnEmptyCollectionInsteadOfNull.cs

```diff
@@ -42,7 +42,7 @@ IEnumerable<char> GetValues(string str)
             if (str == null)
             {
                 return null; // Noncompliant
```

Comment:
The rule does not raise for the following cases:
```
        IEnumerable<string> SwitchExpression(string type)
        {
            return type switch
            {
                "a" => null,
                "b" => default,
            };
        }

        IEnumerable<string> SwitchExpression2(string type) =>
        type switch
        {
            "a" => null,
            "b" => default,
        };
```
Please document that fact and we can also open an issue for that.